rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
+end
```

Comment:
In the Rental class, the rental should add itself to the book and to the person, in order to achieve the belongs_to relation correctly ✔😀✔

please check this part in the lessons: 
![image](https://user-images.githubusercontent.com/87448628/178070982-b0e21e37-8206-4f56-902e-5fdb2bc2d26f.png)